label_studio/ml_models/models.py

```diff
@@ -60,6 +63,16 @@ def full_title(self):
 
     prompt = models.TextField(_('prompt'), null=False, blank=False, help_text='Prompt to execute')
 
+    def delete_predictions(self):
+        """
+        Deletes any predictions that have originated from a ModelVersion
+
+        Currently assumes that we are setting the `model_version` field of Prediction
+        to ModelVersion.full_title
+        """
+
+        predictions = Prediction.objects.filter(model_version=self.full_title).delete()
```

Comment:
do we need to have check here that modelrun status is completed to allow delete on predictions?